config/mkpj.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: mkpj.sh --job=foo ...
+#
+# Arguments to this script will be passed to a dockerized mkpj
+#
+# Example Usage:
+# config/mkpj.sh --job=post-test-infra-push-bootstrap | kubectl create -f -
+# (type "master" at the Base ref prompt)
+
+root="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+config="${root}/config/prow/config.yaml"
+job_config_path="${root}/config/jobs"
+
+docker run -i --rm -v "${PWD}:${PWD}" -v "${root}:${root}" -w "${PWD}" gcr.io/k8s-prow/mkpj "--config-path=${config}" "--job-config-path=${job_config_path}" "$@"
```

Comment:
I don't think you need to mount `${PWD}` or set it as the working dir.

Reply:
Depending on the flavor of Linux and your settings, the volumes *might* need `-v a:b:z`